site/themes/getenvoy/layouts/partials/quickstart.html

```diff
@@ -2,9 +2,7 @@
   <div id="quickstart">
     <div id="quickstart-cli">
       <h2>Fetch, run and debug Envoy using the CLI</h2>
-      <pre>
-        <code class="language-sh">$ curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin <br />$ getenvoy run standard:{{.Site.Params.about.envoyVersion}} -- --version</code>
-      </pre>
+      <pre><code class="language-sh">$ curl -L https://getenvoy.io/cli | bash -s -- -b /usr/local/bin <br />$ getenvoy run standard:{{.Site.Params.about.envoyVersion}} -- --version</code></pre>
```

Comment:
can we keep the indentation?
```
<pre>
    <code>...</code>
</pre>
```

Reply:
If you add enter, you will have `whitespace` problem, 

<img width="1280" alt="Screen Shot 2020-07-13 at 07 35 43" src="https://user-images.githubusercontent.com/1058672/87260383-d04fe400-c4db-11ea-9ecd-422e6a17be9e.png">

You can compare to https://www.getenvoy.io/install/ on `Installation` section btw

<img width="1280" alt="Screen Shot 2020-07-13 at 07 39 13" src="https://user-images.githubusercontent.com/1058672/87260422-01301900-c4dc-11ea-9a22-73405552e0f3.png">

Reply:
Ah yes. actually, this space is part of the design decision, so please keep it as intended :)

Reply:
done, with additional css